.vitepress/config.ts

```diff
@@ -6,21 +6,21 @@ import { defaultGroupLink, sidebarLinks } from '../docs/links';
 
 dotenv.config();
 
-const BASE = '/';
-const BASE_WITH_ORIGIN = `https://developer.stackblitz.com${BASE}`;
+const BASE_PATH = '/';
```

Comment:
💅 Unrelated cosmetic changes: renamed a constant, and tweaked formatting (running Prettier).